packages/apollo-server-graphql-api/server/index.js

```diff
@@ -0,0 +1,5 @@
+const { server } = require("./app");
+
+server.listen(3000).then(({ url }) => {
+    console.log(`🚀  Server ready at ${url}`);
+});
```

Comment:
This file is never being used, right?

Reply:
yes

Reply:
I prefer removing it, in order to keep this sample as simple as possible. Ok?